internal/cmd/auth/update-kubeconfig.go

```diff
@@ -0,0 +1,94 @@
+package auth
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+)
+
+func updateKubeconfigCmd() *cobra.Command {
+	var kubeconfig, baseURL, projectName string
+
+	cmd := &cobra.Command{
+		Use:   "update-kubeconfig",
+		Short: "Update the kubeconfig file",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Determine kubeconfig path
+			var kubeconfigPath string
+			if kubeconfig != "" {
+				kubeconfigPath = kubeconfig
+			} else if envKC := os.Getenv("KUBECONFIG"); envKC != "" {
+				kubeconfigPath = envKC
+			} else {
+				kubeconfigPath = clientcmd.RecommendedHomeFile
+			}
+
+			serverURL, err := url.Parse(baseURL)
+			if err != nil {
+				return fmt.Errorf("failed to parse base URL option: %w", err)
+			}
+
+			if projectName != "" {
+				serverURL.Path = "/control-plane/v1alpha/projects/" + projectName
+			}
+
+			// Load existing config
+			cfg, err := clientcmd.LoadFromFile(kubeconfigPath)
+			if errors.Is(err, os.ErrNotExist) {
+				cfg = api.NewConfig()
+			} else if err != nil {
+				return fmt.Errorf("unable to load kubeconfig from %s: %v", kubeconfigPath, err)
+			}
+
+			clusterName := "datum-cloud"
```

Comment:
I think we're just going with `datum` as the short term for Datum Cloud, right?

Reply:
Updated